php-frontend/src/main/java/org/sonar/plugins/php/api/tree/expression/ArrayAccessTree.java

```diff
@@ -24,16 +24,24 @@
 
 import javax.annotation.Nullable;
 
+/**
+ * <a href="http://php.net/manual/en/language.types.array.php">Array</a> Access
+ *
+ * <pre>
+ *   {@link #object()} [ {@link #offset()} ]
+ *   {@link #object()} { {@link #offset()} }
+ * </pre>
+ */
 @Beta
 public interface ArrayAccessTree extends ExpressionTree {
 
   ExpressionTree object();
 
-  SyntaxToken openBracketToken();
+  SyntaxToken openBraceToken();
 
   @Nullable // FIXME martin: to check
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Take the required action to fix the issue indicated by this comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1134)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this trailing comment on the previous empty line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3ATrailingCommentCheck)